homeassistant/components/hdmi_cec/__init__.py

```diff
@@ -66,7 +68,7 @@
     5: ICON_AUDIO,
 }
 
-CEC_DEVICES = defaultdict(list)
+CEC_DEVICES = defaultdict(list)  # type: ignore
```

Comment:
Why the `ignore` here?

Reply:
Because mypy complains and it is not possible to solve.

Reply:
It does not seem to be used, but I did dare just to remove it.

Reply:
When I remove it I get:
homeassistant/components/hdmi_cec/__init__.py:71: error: Need type annotation for "CEC_DEVICES"  [var-annotated]
but the type in unknown hence the ignore.

Reply:
Why is the type unknown? We should endeavor to actually type things correctly and only ignore as a last resort.

Reply:
Because I have no idea what goes into that dict, as I wrote earlier I cannot find a place where it is used....but seems wrong just to delete it.